packages/dev-server-hmr/src/hmrClientScript.ts

```diff
@@ -0,0 +1,141 @@
+export const hmrClientScript = `
+export const webSocket = ('WebSocket' in window) ?
```

Comment:
Why do we need to create our own websocket connection? You can import the websocket instance from the existing websocket client. 

This way we're creating two websocket connections at runtime.

Reply:
pretty much so we can specify a protocol. we can avoid this by instead prefixing/well naming our message types (i.e. `hmr-hot-accept` or something)